iidm/iidm-api/src/test/java/com/powsybl/iidm/network/util/SVTest.java

```diff
@@ -414,11 +414,16 @@ void testDcLine() {
         SV svA2 = svA1.otherSide(line);
```

Comment:
Can you add a unit test on a real network with a Boundary.getP() ?

Reply:
Done. Adding the test i found a bug in the Dc approximation. The reactance must be used, expressed in per unit.